torch/nn/modules/padding.py

```diff
@@ -67,7 +69,7 @@ class ConstantPad1d(_ConstantPadNd):
                  [ 3.5000,  3.5000,  3.5000, -3.6372,  0.1182, -1.8652,  3.5000]]])
 
     """
-    padding: _size_2_t
+    padding: Tuple[int, int]
```

Comment:
good catch!

Reply:
Before merging the  change, please check that this code is still JIT compilable after the change and scalar to tuple promotions work as expected? (`_size_N_t` are special types which get automatic promotions from integer to list)
Requesting changes to just mention in the PR description which test covers that torch.nn.modules.padding is JITable after the change

Reply:
actually in this particular class padding is always set by 
```
self.padding = _pair(padding)
```
which always return a tuple of 2